transfermethodui/src/androidTest/java/com/hyperwallet/android/ui/transfermethod/SelectTransferMethodTest.java

```diff
@@ -150,6 +180,21 @@ public void testSelectTransferMethod_verifyCountrySelectionList() {
 
         onView(allOf(withId(R.id.country_name), withText("Canada"))).perform(click());
         onView(withId(R.id.select_transfer_method_country_value)).check(matches(withText("Canada")));
+
+        verify(mHyperwalletInsight, atMost(1)).trackClick(any(Context.class), mPageNameCaptor.capture(),
```

Comment:
I'd rather leave the existing tests as they are, not enhancing them to check insights/analytics aspects and instead of creating a specifically for insights where we wouldn't check ui aspects. in this case I'd create a new test case 
testSelectTransferMethod_verifyTrackClick or something like that

Reply:
Sure. Look like @skoong bring up with new class for testing the Insight event, i will update the same.